docs/guides/auth/built_in_ui.rst

```diff
@@ -0,0 +1,241 @@
+.. _ref_guide_auth_built_in_ui:
+
+===========
+Built-in UI
+===========
+
+:edb-alt-title: Integrating EdgeDB Auth's built-in UI
+
+To use the built-in UI for EdgeDB Auth, enable the built-in Auth UI by clicking
+the "Enable UI" button under "Login UI" in the configuration section of the
+EdgeDB UI. Set these configuration values:
+
+-  ``redirect_to``: Once the authentication flow is complete, EdgeDB will
+   redirect the user’s browser back to this URL in your application’s
+   backend.
+-  ``redirect_to_on_signup``: If this is a new user, EdgeDB will redirect
+   the user’s browser back to this URL in your application’s backend.
+-  ``app_name``: Used in the built-in UI to show the user the
+   application’s name in a few important places.
+-  ``logo_url``: If provided, will show in the built-in UI as part of the
+   page design.
+-  ``dark_logo_url``: If provided and the user’s system has indicated
+   that they prefer a dark UI, this will show instead of ``logo_url`` in
+   the built-in UI as part of the page design.
+-  ``brand_color``: If provided, used in the built-in UI as part of the
+   page design.
+
+
+Configuring
+===========
+
+We will demonstrate the various steps below by building a NodeJS HTTP server in
+a single file that we will use to simulate a typical web application. **Note**:
+We are in the process of publishing helper libraries that you can use with
+popular languages and web frameworks. The details below show the inner workings
+of how data is exchanged with the Auth extension from a web app using HTTP. You
+can use this as a guide to integrate with your application written in any
+language that can send and receive HTTP requests.
```

Comment:
Once the first library launches, maybe we can add another admonition (`warning` type) that says something about users of the respective framework (Next.js, I assume) using that instead of trying to integrate from scratch.

```suggestion
a single file that we will use to simulate a typical web application.

.. note::

    We are in the process of publishing helper libraries that you can use with
    popular languages and web frameworks. The details below show the inner
    workings of how data is exchanged with the Auth extension from a web app
    using HTTP. You can use this as a guide to integrate with your application
    written in any language that can send and receive HTTP requests.
```

Reply:
> users of the respective framework (Next.js, I assume) using that instead of trying to integrate from scratch.

Yeah, and I think we should have the docs for each of those frameworks in the docs here so we can link directly from that note.